common/misc.go

```diff
@@ -0,0 +1,29 @@
+package common
+
+import (
+	"os/exec"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+func HardResetHNS() error {
+	log.Debugln("Removing container networks")
+	if err := exec.Command("powershell", "/C", "powershell", "Get-ContainerNetwork", "|",
```

Comment:
Theoretically HNS wrapper should already do this but using it would break test isolation a little. And I don't know how to start/stop hns not from command line, so might as well use this command, too.